scripts/tests.local.sh

```diff
@@ -15,6 +15,7 @@ docker-compose \
     -f docker-compose.shared.depends.yml \
     -f docker-compose.shared.env.yml \
     -f docker-compose.dev.build.yml \
+    -f docker-compose.dev.command.yml \
```

Comment:
Minor cleanup item: I just happened to notice this when reviewing, but perhaps we can delete the commented L27 with this addition?